protocol/x/feetiers/keeper/keeper.go

```diff
@@ -47,12 +48,26 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 
 func (k Keeper) InitializeForGenesis(ctx sdk.Context) {}
 
+// SetVaultKeeper sets the `VaultKeeper` reference in `FeeTiersKeeper`.
+// The reference is set with an explicit method call rather than during `NewKeeper`
+// due to the circular dependency `Clob` -> `Vault` -> `FeeTiers` -> `Clob`.
+func (k *Keeper) SetVaultKeeper(vk types.VaultKeeper) {
+	k.vaultKeeper = vk
+}
+
 func (k Keeper) getUserFeeTier(ctx sdk.Context, address string) (uint32, *types.PerpetualFeeTier) {
+	// Invariant: we know there is at least one tier and that the first tier has no requirements
```

Comment:
nit: Is this invariant comment meant for this line or for the `idx := uint32(0)` line?

Reply:
Yeah. updated these invariant comments